### DISCUSSION TITLE
投递游戏 夏日狂想曲和冬日狂想曲

### DISCUSSION THREAD

**Discussion by Parollel** [Gamuxorg/bbs#440] [投递和请求游戏]
https://store.steampowered.com/app/1227890
https://store.steampowered.com/app/2495450
我想投递这两款游戏，但是没有合适的长期有效文件分享方式。
两款游戏都是用RPGMaker制作的，可以用Linux原生nwjs替换游戏自带的nwjs即可原生运行。
有的问题就是游戏代码中的文件名和一些实际文件名大小写不同，在ntfs上大小写不敏感所以不会报错，在绝大多数Linux文件系统上会报错，我已修复部分（用符号链接）。

**Comment by hyh19962008** [comment]
你好，感谢投稿。方便临时上传到任意网盘里吗？也可以使用文叔叔等一次性的文件中转服务，我们稍后转存到站内服务器上

**Reply by Parollel** [reply]
https://tmp.link/f/66ca0ea6a44e6
游戏启动方式是解压后在 `summer.memories` 或 `winter.memories` 下运行
`./nw .`

**Comment by hyh19962008** [comment]
H游不方便放站里，链接丢这里了
https://www.linuxgame.cn/download/summer_winter_memories.tar.xz
